hms-app/src/components/RelevantIdeasLoader.tsx

```diff
@@ -4,30 +4,30 @@ import { Hackathon, Idea } from '../common/types'
 import { getIdeaDetails } from '../actions/IdeaActions'
 
 type Props = {
-  selectedHackweek: string
+  selectedHackathonID: string
   setRelevantIdea: (relevantIdeaList: Idea[]) => void
   setHackathon: (hackathonData: Hackathon) => void
   setLoading: (boolean: boolean) => void
 }
 
 export default function RelevantIdeasLoader({
-  selectedHackweek,
+  selectedHackathonID,
   setRelevantIdea,
   setHackathon,
   setLoading,
 }: Props) {
-  const [relevantIdeaList, setRelevantIdeaList] = useState([] as Idea[])
+  const [relevantIdeaList, setRelevantIdeaList] = useState<Idea[]>([])
   const [isLoading, setIsLoading] = useState(true)
-  const [hackathonData, setHackathonData] = useState({
+  const [hackathonData, setHackathonData] = useState<Hackathon>({
```

Comment:
You can use custom hook `useLocalStorage("key", defaultValue);` to skip manual checks for the presence of a value :D

Reply:
cant make this work sry